antenna/configlib.py

```diff
@@ -91,6 +101,22 @@ def parse_bool(val):
     raise ValueError('%s is not a valid bool value' % val)
 
 
+def parse_class(val):
+    """Parses a string, imports the module and returns the class
+
+    >>> parse_class('hashlib.md5')
+
+    """
+    module, class_name = val.rsplit('.', 1)
+    module = importlib.import_module(module)
+    try:
+        return getattr(module, class_name)
+    except AttributeError:
+        raise ValueError('%s is not a valid member of %s' % (
```

Comment:
Why a `ValueError`? What was wrong with the `AttributeError`? Also, might it be of interest to re-wrap it as `ImportError`?